working/1426-extension-types/feature-specification.md

```diff
@@ -0,0 +1,720 @@
+# Extension Types
+
+Author: eernst@google.com
+
+Status: Draft
+
+## Change Log
+
+2021.02.15
+  - Initial version, based on 
+    [language issue 1426](https://github.com/dart-lang/language/issues/1426).
+
+## Introduction
+
+This specifies support for extension types in Dart as an enhancement of the
```

Comment:
Either "quote" or *italicize* "extension types" here. It's the first mention of the concept.
Probably quote, since you're not defining it. 

Alternatively, add more text like
> a feature, "extension types", ...

Reply:
I've adjusted the wording to make it clearer that the first sentence is just mentioning the topic of extension types in order to set the scene. The next paragraph is an informal definition of the concept, and it uses `_extension type_`.

We have been using the syntax that indicates that a particular phrase is being defined in both an informal and a formal context. I think that makes sense, especially when there will be two entries in the index, and the reader will be able to find both the informal and the formal "defining occurrence". In this case the informal one is in the introduction, and the formal one occurs in the 'Static analysis' section.